MycroftAI/mycroft-core
# Shellcheck scripts folder

## Description
- Make shell scripts in "scripts" folder pass shellcheck test
- Add scripts folder to CICD shellcheck test
- Remove unused files in scripts folder
- Update my-info.sh to work with current setup (it was working of some very old setup when start-mycroft.sh used screen)

## How to test
- Ensure CICD shellcheck passes
- Ensure the dev_setup.sh works as expected when installing mimic

## Contributor license agreement signed?
CLA [Yes]
